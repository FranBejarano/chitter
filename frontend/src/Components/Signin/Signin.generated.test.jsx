import React from "react";
import renderer from 'react-test-renderer';
import Signin from "./Signin";

jest.mock("axios");
jest.mock("react-redux");
jest.mock("../../redux/userSlice");
jest.mock("react-router-dom");

const renderTree = tree => renderer.create(tree);
describe('<Signin>', () => {
  it('should render component', () => {
    expect(renderTree(<Signin
    />).toJSON()).toMatchSnapshot();
  });

});