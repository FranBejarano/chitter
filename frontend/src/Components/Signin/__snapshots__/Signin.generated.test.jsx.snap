// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Signin> should render component 1`] = `
<form
  className="bg-gray-200 flex flex-col rounded-lg w-max mx-auto gap-10 pb-10"
>
  <h2
    className="text-3xl font-bold text-center"
  >
    Sign in to Chitter
  </h2>
  <input
    className="text-xl py-2 rounded-full px-4"
    onChange={[Function]}
    placeholder="username"
    type="text"
  />
  <input
    className="text-xl py-2 rounded-full px-4"
    onChange={[Function]}
    placeholder="password"
    type="password"
  />
  <button
    className="text-xl py-2 rounded-full px-4 bg-blue-500 text-white"
    onClick={[Function]}
  >
    Sign in
  </button>
  <p
    className="text-center text-xl"
  >
    Don't have an account?
  </p>
  <input
    className="text-xl py-2 rounded-full px-4"
    onChange={[Function]}
    placeholder="name"
    type="text"
  />
  <input
    className="text-xl py-2 rounded-full px-4"
    onChange={[Function]}
    placeholder="username"
    type="text"
  />
  <input
    className="text-xl py-2 rounded-full px-4"
    onChange={[Function]}
    placeholder="email"
    required={true}
    type="email"
  />
  <input
    className="text-xl py-2 rounded-full px-4"
    onChange={[Function]}
    placeholder="password"
    type="password"
  />
  <button
    className="text-xl py-2 rounded-full px-4 bg-blue-500 text-white"
    onClick={[Function]}
    type="submit"
  >
    Sign up
  </button>
</form>
`;
